tests/unit_tests/data_steward/validation/participants/writers_test.py

```diff
@@ -86,21 +100,23 @@ def test_write_to_result_table_error(self, mock_load_csv, mock_upload,
 
         # post conditions
         self.assertEqual(mock_load_csv.call_count, 1)
-        self.assertEqual(mock_upload.call_count, 1)
         self.assertEqual(mock_bucket.call_count, 1)
-
-        upload_path = self.dataset + '/intermediate_results/' + self.site + '.csv'
-        self.assertEqual(
-            mock_upload.assert_called_with(bucket_name, upload_path, ANY), None)
-
-        self.assertEqual(
-            mock_load_csv.assert_called_with(
-                ANY,
-                'gs://' + bucket_name + '/' + upload_path,
-                self.project,
-                self.dataset,
-                self.site + consts.VALIDATION_TABLE_SUFFIX,
-                write_disposition=consts.WRITE_TRUNCATE), None)
+        self.assertEqual(mock_client.get_bucket.call_count, 1)
+        self.assertEqual(mock_client_bucket.blob.call_count, 1)
+        self.assertEqual(mock_client_blob.upload_from_file.call_count, 1)
+
+        upload_path = f'{self.dataset}/intermediate_results/{self.site}.csv'
+        mock_client.get_bucket.assert_called_with(bucket_name)
+        mock_client_bucket.blob.assert_called_with(upload_path)
+        mock_client_blob.upload_from_file.assert_called_with(ANY)
+
+        mock_load_csv.assert_called_with(
+            ANY,
+            'gs://' + bucket_name + '/' + upload_path,
```

Comment:
Same here

Reply:
Yes, I totally forgot. I'll take care of that this morning.